src/event.ts

```diff
@@ -1,12 +1,13 @@
 export interface Event {
     payload?: string;
-    action?: string;
     connKey?: number[];
     dstPort: number;
     rule?: string;
+    handler?: string;
     scanner?: string;
     sensorID: string;
     srcHost: string;
     srcPort: string;
     timestamp: string;
+    decoded?: any;
```

Comment:
Can this type be more specific?

Reply:
They are specific to the protocol. Have a look at what you see in the consol on the web site